### PR TITLE
g/j2: Fill out `game options` menu

### PR DESCRIPTION
**Pull request by xTVaser**
https://github.com/open-goal/jak-project/assets/13153231/b103963e-7fad-4578-b1ae-5042dbdccc99

### CODE REVIEW

**Comment by xTVaser** [comment]
`preserve-this` removal is intentional, it was within a custom file, something that will never be automatically removed by decompilation.

File: goal_src/jak2/pc/progress/progress-static-pc.gc [hunk 1]
```diff
@@ -55,18 +294,40 @@
...
+          (commit-to-file *pc-settings*))))))
 
 
+;; TODO - migrate this menu as well
+;; graphic options
+;;   resolution
+;;   display mode
+;;   display select (disable if windowed)
+;;   vsync
+;;   aspect ratio
```
Comment (water111) [inline goal_src/jak2/pc/progress/progress-static-pc.gc:322]: I think some of these are already there (vsync, full screen)

**Review by water111 — COMMENTED** [review]
Looks good to me!

One minor thing I found during testing is that some of the text is cut off when using 16x9 aspect ratio:
image

File: goal_src/jak2/pc/progress/progress-static-pc.gc (hunk 1 above)
Comment (xTVaser) [reply goal_src/jak2/pc/progress/progress-static-pc.gc:322]: Yeah, this is just the full jak 1 menu layout for reference.

**Comment by xTVaser** [comment]
You're using the non ps2 aspect ratio I assume?  I havn't done a ton of testing outside the ps2 aspect ratios and I had some issues using `adjust-game-x` for that menu, and the sliders so I'm not surprised.

**Pull request merged**